common/pom.xml

```diff
@@ -40,6 +40,9 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>3.0.0-M1</version>
+                <configuration>
+                    <failOnError>false</failOnError>
+                </configuration>
```

Comment:
This fixes issues where cloning the repo and attempting to do a clean install would fail

Reply:
https://stackoverflow.com/questions/49460751/execution-attach-javadocs-of-goal-org-apache-maven-pluginsmaven-javadoc-plugin?noredirect=1

Reply:
Oh, I was about to release 0.17.0 - resetting the version change number. Is there anything I can help you with?

Reply:
(OT 0.32.0 is now being used in `master` though ;) )

Reply:
Hey @carlosalberto How are you? I actually updated to be 0.17.1 after merging your changes

Reply:
@carlosalberto If you're familiar with the tracer code, a review would be great!

Reply:
I found that upgrading the maven-javadoc-plugin from 3.0.0-M1 to 3.1.0 would fix the issue. Possibly better than ignoring failure?

Reply:
Okay I can do that, that's fine by me

Reply:
Although it's a very tertiary failure, but yes agreed